shader/writeback.wgsl

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT OR Unlicense
```

Comment:
Since #456 we have standard copyright headers. Please update this shader file to be consistent with the others.

```suggestion
// Copyright 2023 the Vello Authors
// SPDX-License-Identifier: Apache-2.0 OR MIT OR Unlicense
```